org.eclipse.ice.bats/src/test/java/gov/ornl/rse/tests/bats/ITDataSet.java

```diff
@@ -0,0 +1,192 @@
+/******************************************************************************
+ * Copyright (c) 2019- UT-Battelle, LLC.
+ * All rights reserved. This program and the accompanying materials are made 
+ * available under the terms of the Eclipse Public License v1.0,
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Initial API and implementation and/or initial documentation - 
+ *   Jay Jay Billings
+ *****************************************************************************/
+package org.eclipse.ice.tests.bats;
+
+import static org.junit.Assert.*;
+
+import java.util.UUID;
+
+import org.apache.jena.query.Dataset;
+import org.apache.jena.query.ReadWrite;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdfconnection.RDFConnectionFuseki;
+import org.apache.jena.rdfconnection.RDFConnectionRemoteBuilder;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import org.eclipse.ice.bats.DataSet;
+
+/**
+ * This is a simple test of the BATS Dataset class. It requires that the Fuseki
+ * is running locally on port 3030. The simplest way to do this is to execute
+ * the docker-compose.yml file that is in the directory root BATS directory.
+ * 
+ * @author Jay Jay Billings
+ *
+ */
+public class ITDataSet {
```

Comment:
Did the name of this class change? Wouldn't it normally be something like DataSetTest?

Reply:
Yes it did!
Since we interact with a live Fuseki server, I set this as an integration test.
Specifically, I am using the `maven-failsafe-plugin`.
To pick up this as an integration test, I am following the [plugin's docs](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html), which show under "Inclusions" the regexes used to pick up class names. 

To help make it clear why that name, I copied this from the docs:
> By default, the Failsafe Plugin will automatically include all test classes with the following wildcard patterns:
> -    "**/IT*.java" - includes all of its subdirectories and all Java filenames that start with "IT".
> -    "**/*IT.java" - includes all of its subdirectories and all Java filenames that end with "IT".
> -    "**/*ITCase.java" - includes all of its subdirectories and all Java filenames that end with "ITCase".

Assuming the `IT` is for `I`integration `T`est

Reply:
Wow, they completely ignored the conventions! Ok, thanks.